modules/collection/programs/starship.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkOption mkPackageOption;
+
+  toml = pkgs.formats.toml {};
+
+  cfg = config.programs.starship;
+in {
+  options.programs.starship = {
+    enable = mkEnableOption "starship module.";
+    package = mkPackageOption pkgs "starship" {};
+    settings = mkOption {
+      inherit (toml) type;
```

I think i'd rather you just do
```suggestion
      type = toml.type;
```
for the sake of transparency. idk, lemme know what you think

---

I'm okay with it :)